package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "instabug-reactnative",
-  "version": "1.1.6",
+  "version": "1.1.8",
```

Comment:
Please keep the version number as it was, Since we are going to pump it later on.